Sources/ARKit-CoreLocation/Nodes/PolylineNode.swift

```diff
@@ -34,9 +34,15 @@ public class PolylineNode {
         self.altitude = altitude
         self.boxBuilder = boxBuilder ?? Constants.defaultBuilder
 
+		super.init(location: nil)
```

Comment:
Can you remove the extra whitespace here?

Reply:
I see you beat me to it.  I'll resolve conflicts, thanks!